recipes/lksctp-tools/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, chdir, copy, rm, rmdir, collect_libs
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class LKSCTPToolsConan(ConanFile):
+    name = "lksctp-tools"
+    settings = "os", "arch", "compiler", "build_type"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sctp/lksctp-tools"
+    description = "The Linux Kernel Stream Control Transmission Protocol (lksctp) project"
+    package_type = "library"
+    topics = ("sctp")
+    license = "GPL-2.0, LGPL-2.1"
```

Comment:
```suggestion
    license = ("GPL-2.0", "LGPL-2.1")
```

Each license should be a separated string, so we can create a list.